api.py

```diff
@@ -7,7 +7,7 @@
 
 import detection
 from detection import DetectionData, DetectionSettings
-from utils.custom_logger import CustomLogger
+from utils.custom_logger import main_logger
```

Comment:
The import seems a bit odd. Maybe we can find a better name.

Reply:
Do you mean the `custom_logger` part?

Reply:
What about `utils.logging`, so `from utils.logging import main_logger`

I think the `main_logger` is fine, but if you have suggestions lmk

Reply:
Better